Makefile

```diff
@@ -426,8 +426,11 @@ terratest: # Run terratest suite
 		echo "$(RED)Make sure you run the tests against at least two running clusters$(NC)" ;\
 		exit 1;\
 	fi
-	cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS)
-
+	mkdir -p /tmp/terratest && cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS) | tee /tmp/terratest/all.log
+	curl --location --silent --fail --show-error -o terratest_log_parser https://github.com/gruntwork-io/terratest/releases/download/v0.38.8/terratest_log_parser_linux_amd64
```

Comment:
downloading the file all the time isn't optimal on the other hand, having something like [this](https://github.com/k8gb-io/k8gb/blob/master/olm/generate.sh#L17) in here would make it even more complicated. What about doing only the ` | tee ..` & `mkdir ..` part here and the rest in the gh action?
Btw. a platform specific version is hard-coded (I am developing on mac for instance) -> another conditional or Makefile magic... having this hidden in the action yaml would be better imho